com.unity.multiplayer.mlapi/Tests/Runtime/RpcQueueTests.cs

```diff
@@ -78,33 +47,160 @@ public IEnumerator RpcQueueUnitTest()
                 yield return new WaitForSeconds(0.02f);
 
                 testsAreComplete = rpcPipelineTestComponent.IsTestComplete();
-                exceededMaximumStageIterations = rpcPipelineTestComponent.ExceededMaxIterations();
+                Assert.IsFalse(rpcPipelineTestComponent.ExceededMaxIterations());
             }
+            var testsAreValidated = rpcPipelineTestComponent.ValidateUpdateStages();
 
-            if (!exceededMaximumStageIterations)
-            {
-                testsAreValidated = rpcPipelineTestComponent.ValidateUpdateStages();
-            }
+            //Stop testing
+            rpcPipelineTestComponent.EnableTesting = false;
+
+            Debug.Log($"Exiting status => {nameof(testsAreComplete)}: {testsAreComplete} - {nameof(testsAreValidated)}: {testsAreValidated} -{nameof(exceededMaximumStageIterations)}: {exceededMaximumStageIterations}");
+
+            Assert.IsTrue(testsAreComplete && testsAreValidated);
+
+            //Disable this so it isn't running any longer.
+            rpcPipelineTestComponent.gameObject.SetActive(false);
+
+            yield return null;
```

Comment:
no need for this in this block, we're already `yield`ing in the while-loop above.
```suggestion
```